scripts/release-notes/tests/test_aggregate_release_notes.py

```
@@ -0,0 +1,873 @@
+        buckets = {
+            "total_done": 7,
+            "with_note_customer": 3,
+            "with_note_internal": 0,
+            "skipped": 2,
+            "no_h2_no_skip": 2,
+            "affects_version_empty_alone": 1,
+            "affects_version_unknown": 1,
+        }
+        report = render_completeness_report(buckets, "Community 5.0.4")
+        lines = report.split("\n")
+        table_rows = [l for l in lines if l.strip().startswith("|") and "---" not in l]
```

Ruff (0.15.14)

[error] Ambiguous variable name: `l`

(E741)